storage/storage-proto/src/proto/storage.proto

```diff
@@ -120,12 +120,15 @@ message StartupInfo {
     // start up sync.
     types.LedgerInfoWithSignatures ledger_info = 1;
 
+    // The latest LedgerInfo that carries a ValidatorSet
+    types.LedgerInfoWithSignatures ledger_info_with_validators = 2;
```

Comment:
Love it. Is also going to simplify the code in https://github.com/libra/libra/pull/1819